database/migrations/2023_11_21_163810_create_flights_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('flights', function (Blueprint $table) {
+            $table->id();
+            $table->string('flight_number', 255)->unique();
+            $table->foreignId('airline_id')->constrained()->cascadeOnDelete();
+            $table->foreignId('origin_city_id')->constrained('cities')->cascadeOnDelete();
+            $table->foreignId('dest_city_id')->constrained('cities')->cascadeOnDelete();
+            $table->dateTimeTz('departure_at');
+            $table->dateTimeTz('arrival_at');
```

Comment:
let's just use `->dateTime`. We should always use dateTime and consider it UTC, unless the use case requires to have TZ on DB